src/main/java/com/st/lms/dto/BkCopiesDTO.java

```diff
@@ -0,0 +1,42 @@
+package com.st.lms.dto;
+
+import com.st.lms.models.Author;
+import com.st.lms.models.Book;
+import com.st.lms.models.BookCopies;
+
+public class BkCopiesDTO {

```

Comment:
As with the lack of documentation I'm willing to restrain my disapproval for now, but for me to give my :+1: to any future major PR (including initialization of any next-generation new-project repos) it should include refactoring of entities to refer to each other by object references rather than storing only `int` IDs of the other objects they relate to. (I say this here because once you've changed your design to do the obviously correct thing, these "DTOs" will for the most part no longer be needed.)